waltid-wallet-api/src/main/kotlin/id/walt/webwallet/service/WalletKitWalletService.kt

```diff
@@ -170,9 +170,9 @@ class WalletKitWalletService(tenant: String, accountId: UUID, walletId: UUID) :
             )
```

Comment:
I can't see where this class is used. The only place that references it is in the `getWalletService` of WalletServiceManager and that line is commented out.